docs/_pages/grammar.md

```diff
@@ -0,0 +1,77 @@
+---
+permalink: /grammar
+title: Grammar
+toc: true
+---
+
+This is the complete syntax grammar for Luau in EBNF. More information about the terminal nodes String and Number
+is available in the [syntax section](syntax).
+
+```
+chunk ::= {stat [`;']} [laststat [`;']]
+block ::= chunk
+stat ::= varlist `=' explist |
```

Comment:
I don't think a `bindinglist` works here, since a type annotation isn't permitted in this location, and bindinglist only represents a NAME identifier, but in general assignment we can have a complex var (`a[foo()] = 1`).

But yeah, there is no `var`/`varlist`, need to add that in

Reply:
Ah, you're right. I missed the lack of local keyword.